worker-languagedetection-container/testcases/automated/CAF_4001/afrikaans.yaml.testcase

```diff
@@ -1,9 +1,9 @@
 ---
 tag: "CAF_4001/afrikaans.yaml"
 testCaseInformation:
-  associatedTickets: CAF-414
+  associatedTickets: null
```

Comment:
These lines should not have been changed on any of the release testcases.

Reply:
Sorry, I regenerated testcases and forgot to undo these changes.